### ISSUE TITLE
Screenshots are hidden for embed games

### ISSUE THREAD

**Issue by fasterthanlime** [leafo/itch.io#202]
Find a layout that makes sense?

**Comment by leafo** [comment]
yeah. It seems like an edge case to me, the game is right there so I'm guessing most people would just play it, so making this default does not seem like a good idea.

**Comment by felixplesoianu** [comment]
There are embedded games with a downloadable version, which may look differently, and you want to show people screenshots from it. In fact I hit exactly this case with one of mine, and had to give up providing the download to avoid confusing people.

**Comment by fasterthanlime** [comment]
Adding a video isn't supported on embed pages either, but it should be.

**Comment by ianfitzpatrick** [comment]
Hey just to make this seem less edge casey, we have been discussing this exact issue in our local gamedev slack the last day.

I'm just of the opinion it's a lot to ask a player to play a game right away without giving them some idea of what's up with screenshots + a trailer first, and the embed is too much of an immediate ask.

It would be nice if you could have a button in the description that would reveal the embed in the middle of the page, so you could still have the low resistance threshold of playing on page, without demanding it be the first thing someone does when they hit your game page.

FWIW I also have downloads for my game. I ended up just linking to a web version in the description. See:

https://dadcore.itch.io/sdf

**Comment by VirtualNomad19** [comment]
can Sidebar finally be the Default for Screenshots, please? 9 years of everyone forgetting they have to manually set them visible when this seems like a simple fix/convenience?
